utils/client.js

```diff
@@ -1,7 +1,11 @@
-import ApolloClient from "apollo-boost";
-import { InMemoryCache } from "apollo-cache-inmemory";
+import ApolloClient from 'apollo-boost';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125704)
```suggestion
import ApolloClient from "apollo-boost";
```